Source/FullScreenPokemon.ts

```diff
@@ -4869,7 +4869,7 @@ module FullScreenPokemon {
         partyActivateCheckThing(player: IPlayer, pokemon: IPokemon, move: IMoveSchema): void {
             var borderedThing: IThing = player.bordering[player.direction];
 
-            if (borderedThing && borderedThing.title === move.characterName) {
+            if (borderedThing && borderedThing.title.indexOf(move.characterName) !== -1) {
```

Comment:
Can you put a `@todo` mentioning this should be refactored? A better way would be to have a .hmActivate property or something like that, but that's out of scope for your changes here.